tests/bot/exts/info/test_information.py

```diff
@@ -281,9 +281,13 @@ async def test_create_user_embed_uses_string_representation_of_user_in_title_if_
         """The embed should use the string representation of the user if they don't have a nick."""
         ctx = helpers.MockContext(channel=helpers.MockTextChannel(id=1))
         user = helpers.MockMember()
+        public_flags = unittest.mock.MagicMock()
```

Comment:
```suggestion
        public_flags = unittest.mock.MagicMock(verified_bot=False)
```
This can be passed as a kwarg instead of setting it manually below

Reply:
Changed in 1fdd5aab